### PR TITLE
postgis connection parameter docs update

### PR DESCRIPTION
**Pull request by jodygarnett**
## Description

Noticed a few postgis connection parameters were not described in the user guide.

## Checklist

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)

The following are required only for core and extension modules, while they are welcomed, but not required, for community modules:

- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

**Comment by jodygarnett** [comment]
Can I get a review please? this is a doc only change so please ignore code build failures and focus on content changes.

**Comment by jodygarnett** [comment]
rebased in the hopes of getting of passing automated checks.

File: doc/en/build.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
   <target name="sphinx" if="sphinx.available">
                 <echo message="Running sphinx-build -D release=${project.version} -W -b ${build} -d &quot;${build.directory}/${id}/doctrees&quot; . &quot;${build.directory}/${id}/${build}&quot;"/> 
     <exec executable="sphinx-build" failonerror="true" dir="${basedir}/${id}/source">
-      <arg line="-D release=${project.version} -q -W -b ${build} -d &quot;${build.directory}/${id}/doctrees&quot; . &quot;${build.directory}/${id}/${build}&quot;"/>
+      <arg line="-D release=${project.version} -j auto -q -W --keep-going -b ${build} -d &quot;${build.directory}/${id}/doctrees&quot; . &quot;${build.directory}/${id}/${build}&quot;"/>
```
Comment (jodygarnett) [inline doc/en/build.xml]: these changes make sphinx-build multithreaded and report back all warnings

File: doc/en/user/source/data/database/postgis.rst [hunk 2]
```diff
@@ -15,15 +15,16 @@ Using default connection
...
 
+.. figure:: images/postgis-basic-info.png
+   
    *Adding a PostGIS database*
 
 .. list-table::
    :widths: 20 80
 
-   * - **Option**
+   * - **Basic Store Info**
```
Comment (jodygarnett) [inline doc/en/user/source/data/database/postgis.rst]: use `header-rows` to avoid extra typing

File: doc/en/user/source/data/database/postgis.rst [hunk 3]
```diff
@@ -59,10 +85,63 @@ To begin, navigate to :menuselection:`Stores --> Add a new store --> PostGIS NG`
      - Time (in seconds) the connection pool will wait before timing out.
    * - :guilabel:`validate connections`
      - Checks the connection is alive before using it.
+   * - Evictor run periodicity
```
Comment (jodygarnett) [inline doc/en/user/source/data/database/postgis.rst]: these should use `gui-label` for consistency

File: doc/en/user/source/data/database/postgis.rst [hunk 4]
```diff
@@ -59,10 +86,63 @@ To begin, navigate to :menuselection:`Stores --> Add a new store --> PostGIS NG`
      - Time (in seconds) the connection pool will wait before timing out.
    * - :guilabel:`validate connections`
      - Checks the connection is alive before using it.
+   * - :guilabel:`Evictor run periodicity`
+     - Number of seconds between idle object evictor runs.
```
Comment (bradh) [inline doc/en/user/source/data/database/postgis.rst]: This "evictor" stuff is probably magic to most users - they won't know what to do. We should provide some recommended values or at least general guidance.

File: doc/en/user/source/data/database/postgis.rst [hunk 5]
```diff
@@ -59,10 +86,63 @@ To begin, navigate to :menuselection:`Stores --> Add a new store --> PostGIS NG`
...
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Connection Parameter
+     - Description
    * - :guilabel:`Loose bbox`
      - Performs only the primary filter on the bounding box.  See the section on :ref:`postgis_loose_bbox` for details.
-   * - :guilabel:`preparedStatements`
-     - Enables prepared statements.
+   * - :guilabel:`Estimated extends`
```
Comment (bradh) [inline doc/en/user/source/data/database/postgis.rst]: It matches the screenshot, but I think this should be `Estimated extents`. Another ticket?

**Pull request merged**